tests/integration/targets/custom_image/tasks/tests.yml

```
@@ -0,0 +1,267 @@
+  assert:
+    that:
+      - image_by_name is not changed
+      - image_by_name.uuid == image2.uuid
+
+- name: Update slug of a custom image
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ item }}"
+    slug: ansible-image-slug
+  register: image
+  loop:
+    - "{{ image2.uuid }}"
+- name: Verify update slug of a custom image
+  assert:
+    that:
+      - image is changed
+
+- name: Get custom image with updated slug
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ item }}"
+  loop:
+    - "{{ image2.uuid }}"
+  register: image
+- name: Verify update slug of a custom image
+  assert:
+    that:
+      - image.results[0].slug == "ansible-image-slug"
+
+- name: Update tags of a custom image
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ item }}"
+    tags:
+      project: luna
+  register: image
+  loop:
+    - "{{ image2.uuid }}"
+- name: Verify update tags of a custom image
+  assert:
+    that:
+      - image is changed
+      - image.results[0].tags == "project: luna"
+
+- name: Update user_data_handling of a custom image
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ item }}"
+    user_data_handling: 'extend-cloud-config'
+  register: image
+  loop:
+    - "{{ image2.uuid }}"
+- name: Verify update user_data_handling of a custom image
+  assert:
+    that:
+      - image is changed
+
+- name: Get custom image with updated user_data_handling
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ item }}"
+  loop:
+    - "{{ image2.uuid }}"
+  register: image
+- name: Verify update user_data_handling of a custom image
+  assert:
+    that:
+      - image.results[0].user_data_handling == "extend-cloud-config"
+
+- name: Update slug, tags and user_data_handling of a custom image
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ item }}"
+    slug: yet-another-slug
+    tags:
+      project: jupiter
+    user_data_handling: 'pass-through'
+  loop:
+    - "{{ image2.uuid }}"
+  register: image
+- name: Verify update slug, tags and user_data_handling of a custom image
+  assert:
+    that:
+      - image is changed
+      - image.results[0].slug == "yet-another-slug"
+      - image.results[0].tags == "project: jupiter"
+      - image.results[0].user_data_handling == "pass-through"
+
+- name: List all custom images
+  uri:
+    url: 'https://api.cloudscale.ch/v1/custom-images'
+    headers:
+      Authorization: 'Bearer {{ cloudscale_api_token }}'
+    status_code: 200
+  register: image_list
+  until: image_list is not failed
+  retries: 5
+  delay: 3
+- name: Verify that two custom images are created by this test run
+  assert:
+    that:
+      - image_list.json | selectattr("name","search", "{{ cloudscale_resource_prefix }}" ) | list | length == 2
+
+- name: Delete image by uuid
+  cloudscale_ch.cloud.custom_image:
+    uuid: "{{ image1.uuid }}"
+    state: absent
+  register: image1
+- name: Verify delete image by uuid
+  assert:
+    that:
+      - image1 is changed
+      - image1.state == 'absent'
```

Nitpick: Please insert a blank line here.
```suggestion
      - image1.state == 'absent'
      
```